Resources/Prototypes/Entities/Mobs/NPCs/silicon.yml

```
@@ -184,6 +184,10 @@
     speechVerb: Cluwne
   - type: StepTrigger
     intersectRatio: 0.2
+  - type: StepTriggerImmune # Goobstation
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)